tests/integration/conftest.py

```diff
@@ -46,3 +46,14 @@ def find(bundle_name="bundle-large_whitelist.json"):
             return json.load(f)
 
     return find
+
+
+@pytest.fixture(autouse=True)
+def set_legacy_provider(monkeysession):
+    def _provider_name(section):
+        return "legacy"
```

Comment:
@zburstein Can you open a ticket to investigate changing this to grype? Whether we switch over to doing that completely, or do parallel runs with both providers for now, either way we need to be running these tests with the v2 provider eventually (and probably sooner rather than later).